src/manager.cpp

```diff
@@ -398,8 +398,7 @@ void Manager::collectSingleFruVpd(
 void Manager::deleteSingleFruVpd(
     const sdbusplus::message::object_path& i_dbusObjPath)
 {
-    // Dummy code to supress unused variable warning. To be removed.
-    logging::logMessage(std::string(i_dbusObjPath));
+    m_worker->deleteFruVpd(std::string(i_dbusObjPath));
```

Comment:
It should be checked if there is worker object available or not before using it.

Reply:
Done